test/eventrouter.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This is a test suite for the eventrouter
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+
+function get_es_count() {
+    curl_es "$1" "/_search?size=0&q=$2" | python -c "import sys, json; print json.load(sys.stdin)['hits']['total']"
+}
+
+os::log::info Starting eventrouter test at $( date )
+
+espod=$( get_running_pod es )
+eventcount=$( get_es_count "$espod" 'kubernetes.event.verb:*' )
+
+if [ "$eventcount" == 0 ]; then
+    os::log::error No events from eventrouter processed
+    exit 1 
+fi
```

Comment:
These lines should be replaced with something like this:
```
os::cmd::expect_success_and_not_text "curl_es $espod /_count?q=kubernetes.event.verb:* | get_count_from_json" "^0\$"
```
then you don't need `get_es_count` either.

Reply:
use `/project.*/_count` - make sure we only look at the logs from pods/containers, to avoid any strange or spurious hits